SbtPlugin/src/main/scala/com/dslplatform/sbt/SbtDslPlatformPlugin.scala

```diff
@@ -97,7 +97,7 @@ object SbtDslPlatformPlugin extends AutoPlugin {
     Seq(
       sourceDirectories := Nil,
       unmanagedSources := Nil,
-      sourceGenerators += dslSourcesForLibrary.taskValue,
+      sourceGenerators in Compile += dslSourcesForLibrary.taskValue,

```

Comment:
what about source generators in test?

Reply:
This was just an experiment that is not working out. I need to figure out what is going on, without this sbt just doesn't see generated sources. With this, stuff works but DSL generated sources are compiled twice. First by the DSL sbt plugin, then by sbt itself when building the main api subproject.